build.gradle.kts

```diff
@@ -9,18 +16,44 @@ allprojects {
     mavenCentral()
   }
 
+  apply(plugin = "com.vanniktech.maven.publish")
+
+  // Allow publishing to a private Maven repository, other than to Maven Central
+  val privateRepoUrl = mavenRepositoryUrl()
+
+  if (privateRepoUrl != null) {
+    extensions.configure<PublishingExtension>("publishing") {
+      repositories {
+        maven {
+          name = mavenRepositoryName() ?: "PrivateNexus"
+          url = uri(privateRepoUrl)
+          isAllowInsecureProtocol = true
+        }
+      }
+    }
+  }
 }
 
 subprojects {
-
   tasks.withType<Test>().all {
     testLogging {
       showStandardStreams = true
       showExceptions = true
-      exceptionFormat = org.gradle.api.tasks.testing.logging.TestExceptionFormat.FULL
+      exceptionFormat = TestExceptionFormat.FULL
     }
   }
+}
 
+release {
+  buildTasks = listOf(
+    ":antlr-kotlin-runtime:publishAllPublicationsToMavenCentralRepository",
+    ":antlr-kotlin-target:publishAllPublicationsToMavenCentralRepository",
```

Comment:
Here we may add the task to release the gradle plugin too

Reply:
Are you going to add it? Maybe better if you do it, since you have the Gradle Plugin Portal account.

Reply:
I will do that (this is now #117)